livekit-agents/livekit/agents/voice/avatar/_byte_stream_io.py

```diff
@@ -103,9 +103,9 @@ def clear_buffer(self) -> None:
         task.add_done_callback(self._tasks.discard)
 
 
-class DataStreamAudioReceiver(AudioReceiver):
+class ByteStreamAudioReceiver(AudioReceiver):
```

Comment:
I'm still in favor of AudioStreamReceiver, users don't need to know the underlying transport. Datastream is hidden

Reply:
but it also makes sense that `ByteStreamAudioReceiver` is a specific implementation of `AudioReceiver` ? 
`AudioStreamReceiver` and `AudioReceiver` sound arbitrary/interchangeable